src/Database/Apprenticeship.sql

```diff
@@ -8,3 +8,6 @@
     [DateOfBirth] DATETIME NOT NULL,
     [ApprenticeshipHashedId] NVARCHAR(100) NULL
 )
+GO
+CREATE INDEX IX_Apprenticeship_Uln ON Apprenticeship (Uln)
```

Comment:
Should this index include ApprovalsApprenticeshipId if the aim is to speed up the DB hit you added to check for duplicates?